tools/psa/release.py

```diff
@@ -167,30 +183,30 @@ def build_tests_mbed_spm_platform(target, toolchain, profile):
                                     target, 'test_spec.json'),
         '--build-data', os.path.join(ROOT, 'BUILD', 'tests',
                                      target, 'build_data.json'),
-        '-n', MBED_PSA_TESTS
-    ], stdout=subprocess_output, stderr=subprocess_err)
-
+        '-n', MBED_PSA_TESTS] + args
```

Comment:
The `+ args` portion doesn't need to be in the array?

Reply:
`args` is an array `array + array = array`